src/lib/Sympa/Archive.pm

```diff
@@ -316,8 +316,9 @@ sub html_next {
     unless ($self->{_html_metadatas}) {
         my $dh;
         unless (opendir $dh, $self->{arc_directory}) {
-            die sprintf 'Cannot open dir %s: %s', $self->{arc_directory},
-                $ERRNO;
+            $log->syslog('err', 'Cannot open dir %s: %s', $self->{arc_directory},
+                         $ERRNO);
+            return undef;
```

Comment:
I think this is the case when "Sympa has no choice but to die" (cf. #90 ).  It's fatal that `$self->{arc_directory}` is set, i.e. `select_archive()` has succeeded, while it cannot be opened.

Reply:
I can see your point ... but it leaves the web interface user with the die output which should not happen other than real corner cases. What about checking whether the directory can be read in the beginning of do_arc?

Reply:
Checking readability may not be the solution, I think. But this topic seems beyond the scope of this pr, and I’ll stop commenting for the present.